tip-wlan/charts/wlan-ssc-service/files/cqlshrc.tip-wlan

```diff
@@ -19,12 +19,11 @@
 
 [authentication]
 ;; If Cassandra has auth enabled, fill out these options
-username = cassandra 
+username = cassandra
```

Comment:
Shouldn't this be a template instead of hardcoded user/pass?

Reply:
its always cassandra\cassandra - so probably doesn't make a lot of sense? we don't even have a variable for that

Reply:
@4c74356b41 why it's always cassandra? If for example I will run this with my own instance is cassandra, I'm sure it won't use the default password. Also, the end user can specific a different user via the cassandra chart, in that case, everything here will break.